### DISCUSSION TITLE
How to change growth curve?

### DISCUSSION THREAD

**Discussion by ghost** [Taureon/aps-plus-plus#292] [Help]
How?

**Comment by atvsipi** [comment]
https://github.com/Taureon/aps-plus-plus/blob/e8bf38a3067ee35cb7e13d7aff8d561f143f04f1/server/modules/setup/gamemodeconfigs/growth.js#L3

**Comment by GDenisC** [comment]
i made it but it was bad (and i got the dev role in the discord server at this time and got fired in 1 hour after lol)
